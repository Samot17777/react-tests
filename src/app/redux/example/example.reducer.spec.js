import * as types from './example.action-types';
import {exampleReducer} from "./example.reducer";

describe('Example reducer', () => {

    // using snapshoot

    it('sets number', () => {
        expect(exampleReducer(undefined, {
            type: types.EXAMPLE_ACTION,
            payload: 5
        })).toMatchSnapshot();
    });

    // using strict comparison

    it('sets number', () => {
        expect(exampleReducer(undefined, {
            type: types.EXAMPLE_ACTION,
            payload: 5
        })).toEqual({
            exampleNumber: 5
        });
    });

});