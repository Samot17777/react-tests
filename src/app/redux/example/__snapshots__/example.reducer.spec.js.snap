// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Example reducer sets number 1`] = `
Object {
  "exampleNumber": 5,
}
`;
